LongWarOfTheChosen/Config/XComLW_Overhaul.ini

```diff
@@ -1581,15 +1581,18 @@ BLACK_MARKET_PERSONNEL_INFLATION_PER_FORCE_LEVEL=0.075
 BLACK_MARKET_SOLDIER_DISCOUNT=0.4
 
 [LW_Overhaul.X2EventListener_Headquarters]
-CA_RISK_REDUCTION_PER_RANK[0]=15   ; Rookie
-CA_RISK_REDUCTION_PER_RANK[1]=12   ; Veteran
-CA_RISK_REDUCTION_PER_RANK[2]=10   ; Commander
-CA_RISK_REDUCTION_PER_RANK[3]=10   ; Legend
-
-CA_RISK_INCREASE_PER_FL[0]=2       ; Rookie
-CA_RISK_INCREASE_PER_FL[1]=2       ; Veteran
-CA_RISK_INCREASE_PER_FL[2]=2       ; Commander
-CA_RISK_INCREASE_PER_FL[3]=2       ; Legend
+CA_RISK_REDUCTION_PER_RANK[0]=12   ; Rookie
+CA_RISK_REDUCTION_PER_RANK[1]=10   ; Veteran
+CA_RISK_REDUCTION_PER_RANK[2]=8    ; Commander
+CA_RISK_REDUCTION_PER_RANK[3]=8    ; Legend
+
+CA_RISK_INCREASE_PER_FL[0]=4       ; Rookie
+CA_RISK_INCREASE_PER_FL[1]=4       ; Veteran
+CA_RISK_INCREASE_PER_FL[2]=4       ; Commander
+CA_RISK_INCREASE_PER_FL[3]=4       ; Legend
+
+; The FL at which the failure risk no longer increases
+CA_RISK_FL_CAP=12
```

Comment:
That seems a little low, especially since risk reduction happens per person. I'd make it 14, maybe even 16.

Reply:
Higher than 12 means that a full squad of MSGTs will not have 100% success chance on the hardest covert actions. It's a base chance of failure of 80% + FL12 (4 * 12) = 128%.

Of course, having a small chance of failure for MSGTs on the hardest covert actions isn't necessarily a major problem. I don't really have a strong opinion on this. Could up it to FL16, which would mean a maximum failure chance of 16%. 🤷‍♂

Reply:
Eh, let's try it as is for now, pretty easy to change if it turns out to be boring.